.github/workflows/ruby.yml

```diff
@@ -0,0 +1,112 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+name: Run lint and test
+
+on:
+  push:
+    branches: [ production ]
```

Comment:
Nit: these spaces bother me, can we fix them? On line 13 as well. `branches: [production]`